lib/msal-browser/src/controllers/ControllerFactory.ts

```diff
@@ -27,20 +27,21 @@ export class ControllerFactory {
 
     async createController(): Promise<IController> {
         const standard = new StandardOperatingContext(this.config);
-        const metaOS = new TeamsAppOperatingContext(this.config);
+        const teamsApp = new TeamsAppOperatingContext(this.config);
```

Comment:
Does this refer explicitly to Teams App?

Reply:
For the moment this is just for Teams Apps.  It may be generalized in the future.